.vscode/autogpt-workspace.code-workspace

```diff
@@ -0,0 +1,69 @@
+{
+  "folders": [
+    {
+      "name": ".github",
+      "path": "../.github"
+    },
+    {
+      "name": ".vscode",
+      "path": "../.vscode"
+    },
+    {
+      "name": "assets",
+      "path": "../assets"
+    },
+    {
+      "name": "autogpt",
+      "path": "../autogpt"
+    },
+    {
+      "name": "benchmark",
+      "path": "../benchmark"
+    },
+    {
+      "name": "docs",
+      "path": "../docs"
+    },
+    {
+      "name": "forge",
+      "path": "../forge"
+    },
+    {
+      "name": "frontend",
+      "path": "../frontend"
+    },
+    {
+      "name": "autogpt_server",
+      "path": "../rnd/autogpt_server"
+    },
+    {
+      "name": "root_folder",
+      "path": ".."
+    }
+  ],
+  "settings": {
+    "python.analysis.useLibraryCodeForTypes": true,
+    "python.analysis.autoImportCompletions": true,
+    "python.analysis.typeCheckingMode": "basic",
+    "python.analysis.completeFunctionParens": true,
+    "python.analysis.inlayHints.functionReturnTypes": true,
+    "python.analysis.inlayHints.variableTypes": true,
+    "python.analysis.autoFormatStrings": true,
+    "python.analysis.inlayHints.callArgumentNames": "partial",
+    "python.analysis.fixAll": [
+      "source.convertImportFormat",
+      "source.unusedImports"
```

Comment:
This one is actually quite annoying imo, because if you add an import with the intention of using it later and you hit save, the import disappears.

Reply:
Removing it is fine with me, just make the commits. The workspace splitting and extension recommendations matter here more imo. Each project will probably have its own settings

Reply:
removed